docs/pages/management/admin/labels.mdx

```diff
@@ -1,405 +1,323 @@
 ---
-title: Labels
+title: Add Labels to Resources
 description: Labeling Teleport resources
 ---
 
 Teleport allows you to add arbitrary key-value pairs to applications, servers,
-databases, and other resources in your cluster. You can use these to filter
-resources when making queries via `tctl` and `tsh` as well as to limit the
-resources that Teleport roles can access.
+databases, and other resources in your cluster. You can use labels to do 
+the following:
 
-This guide explains how to add labels to Teleport resources.
+- Filter the resources returned when running `tctl` and `tsh` commands.
+- Limit the resources that Teleport roles can access.
 
-There are two kinds of labels:
+This guide demonstrates how to add labels to enrolled server resources.
+However, you can use these instructions to add labels for any type 
+of resource managed by Teleport.
```

Comment:
There is some nuance in the details here. Different resources get labeled in different ways. Should we soften this sentence a bit? Maybe "you can follow a similar process to label other types of resources"